README.rst

```diff
@@ -71,23 +72,23 @@ How does it work? See below:
 See the documentation for more details and features.
 
 Supported Odoo server versions
-------------------------------
+==============================
 
 `OdooRPC` has been tested on `Odoo` 8.0, 9.0 and 10.0.
```

Comment:
Include 11.0

Reply:
Officially the 11.0 is not supported yet, I'll make another PR for that (to test it on Travis + updating the documentation). The support of Python 3.2 and 3.3 will be dropped too.

Reply:
OK then